packages/suite/src/utils/wallet/cardanoUtils.ts

```diff
@@ -0,0 +1,262 @@
+                      },
+                  ]
+                : [],
+            setMax,
+        };
+    });
+
+export const getShortFingerprint = (fingerprint: string) => {
+    const firstPart = fingerprint.substring(0, 10);
+    const lastPart = fingerprint.substring(fingerprint.length - 10);
+    return `${firstPart}…${lastPart}`;
+};
+
+export const transformUtxos = (utxos: Account['utxo']): types.Utxo[] => {
+    const result: types.Utxo[] = [];
+    utxos?.forEach(utxo => {
+        const foundItem = result.find(
+            res => res.txHash === utxo.txid && res.outputIndex === utxo.vout,
+        );
+
+        if (utxo.cardanoSpecific) {
+            if (!foundItem) {
+                // path: utxo.path,
+                result.push({
+                    // path: utxo.path,
+                    address: utxo.address,
+                    txHash: utxo.txid,
+                    outputIndex: utxo.vout,
+                    amount: [{ quantity: utxo.amount, unit: utxo.cardanoSpecific.unit }],
+                });
+            } else {
+                foundItem.amount.push({ quantity: utxo.amount, unit: utxo.cardanoSpecific.unit });
+            }
+        }
+    });
+
+    return result;
+};
+
+export const prepareCertificates = (certs: CardanoCertificate[]) => {
+    // convert trezor-connect certificate format to cardano coin-selection lib format
+    const convertedCerts: types.Certificate[] = [];
+    certs.forEach(cert => {
+        switch (cert.type) {
+            case CardanoCertificateType.STAKE_DELEGATION:
+                convertedCerts.push({
+                    type: cert.type,
+                    pool: cert.pool!,
+                });
+                break;
+            case CardanoCertificateType.STAKE_REGISTRATION:
+            case CardanoCertificateType.STAKE_DEREGISTRATION:
+                convertedCerts.push({
+                    type: cert.type,
+                });
+                break;
+            // no default
+        }
+    });
+    return convertedCerts;
+};
+
+export const parseAsset = (
+    hex: string,
+): {
+    policyId: string;
+    assetNameInHex: string;
+} => {
+    const policyIdSize = 56;
+    const policyId = hex.slice(0, policyIdSize);
+    const assetNameInHex = hex.slice(policyIdSize);
+    return {
+        policyId,
+        assetNameInHex,
+    };
+};
+
+export const getDelegationCertificates = (
+    stakingPath: string,
+    poolHex: string | undefined,
+    shouldRegister: boolean,
+) => {
+    const result: CardanoCertificate[] = [
+        {
+            type: CardanoCertificateType.STAKE_DELEGATION,
+            path: stakingPath,
+            pool: poolHex,
+        },
+    ];
+
+    if (shouldRegister) {
+        result.unshift({
+            type: CardanoCertificateType.STAKE_REGISTRATION,
+            path: stakingPath,
+        });
+    }
+
+    return result;
+};
+
+export const getTtl = (testnet: boolean) => {
+    const shelleySlot = testnet ? 48409057 : 4924800;
```

Comment:
just like `CARDANO_DEFAULT_TTL` is now imported from constnats, would it make sense also for shelleySlot testnet / mainnet and shellyTimestamp testnet / mainnet to move these values into constants?